modules/apps/batch-engine/batch-engine-test/src/testFunctional/macros/BatchEngine.macro

```diff
@@ -16,18 +16,27 @@ definition {
 		return ${archivedFilePath};
 	}
 
-	macro assertExportedJsontContainsCorrectConfigurationObject {
-		Variables.assertDefined(parameterList = "${jsonObject},${expectedValue}");
+	macro assertExportedFileContainsCorrectObject {
+		Variables.assertDefined(parameterList = "${fileName},${jsonObject},${expectedValue}");
 
-		var fileContent = TestCase.getTempFileContent(fileName = "export.batch-engine-data.json");
+		var fileContent = TestCase.getTempFileContent(fileName = ${fileName});
 
-		var actualValue = JSONUtil.getWithJSONPath(${fileContent}, "$.configuration.${jsonObject}");
+		var actualValue = JSONUtil.getWithJSONPath(${fileContent}, ${jsonObject});
 
 		TestUtils.assertEquals(
 			actual = ${actualValue},
 			expected = ${expectedValue});
 	}
 
+	macro assertExportedJsontContainsCorrectConfigurationObject {
```

Comment:
This method has been renamed to assertExportedJsontContainsCorrectObject, see more details in this pr assertExportedJsontContainsCorrectConfigurationObject.

Reply:
@yanan Can you send the link for PR or it only change the macro name?